src/index.js

```diff
@@ -247,7 +247,36 @@ export type {
 export {
   validate,
   ValidationContext,
+
+  // All validation rules in the GraphQL Specification.
   specifiedRules,
+
+  // Individual validation rules.
+  UniqueOperationNamesRule,
+  LoneAnonymousOperationRule,
+  KnownTypeNamesRule,
+  FragmentsOnCompositeTypesRule,
+  VariablesAreInputTypesRule,
+  ScalarLeafsRule,
+  FieldsOnCorrectTypeRule,
+  UniqueFragmentNamesRule,
+  KnownFragmentNamesRule,
+  NoUnusedFragmentsRule,
+  PossibleFragmentSpreadsRule,
+  NoFragmentCyclesRule,
+  UniqueVariableNamesRule,
+  NoUndefinedVariablesRule,
+  NoUnusedVariablesRule,
+  KnownDirectivesRule,
+  UniqueDirectivesPerLocationRule,
+  KnownArgumentNamesRule,
+  UniqueArgumentNamesRule,
+  ArgumentsOfCorrectTypeRule,
+  ProvidedNonNullArgumentsRule,
+  DefaultValuesOfCorrectTypeRule,
+  VariablesInAllowedPositionRule,
+  OverlappingFieldsCanBeMergedRule,
+  UniqueInputFieldNamesRule,
```

Comment:
I'd alphabetize these to make it obvious where to insert new rules in the future.